apps/docs/src/.vitepress/env.ts

```diff
@@ -6,5 +6,5 @@
  * e.g. when used inside .vitepress/config.ts
  */
 export const getStorybookHost = (env: Record<string, string> = import.meta.env): string => {
-  return env.VITE_STORYBOOK_HOST || "https://onyx-storybook-prod.apps.01.cf.eu01.stackit.cloud";
+  return env.VITE_STORYBOOK_HOST || "https://storybook.onyx.schwarz";
```

Comment:
does that work for you?
<img width="1172" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/a8ef6346-8f06-4f24-8608-7c1a871d7bed">

Reply:
Hmm yes I have the same error but it works on my private laptop. Maybe its a VPN issue?
I'll ask in the IT4YOU ticket

Reply:
I changed it back to the old URL for now so we can merge this PR